src/countryOfSearcher.js

```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+
+const pText = document.getElementById('p-text');
+const article = document.getElementById('article');
+async function reachToSearch(name){
+    article.innerHTML = '';
+    pText.innerHTML = '';
+    try {
+        const nameOfCountry = await axios.get(`https://restcountries.com/v2/name/${name}`);
+        const subregion = nameOfCountry.data[0].subregion;
+        const population = nameOfCountry.data[0].population;
+        const capital = nameOfCountry.data[0].capital;
+        const money = nameOfCountry.data[0].currencies[0].name;
+        const flag = nameOfCountry.data[0].flag;
+        const name1 = nameOfCountry.data[0].name;
```

Comment:
Je zou deze iets verder kunnen versimpelen zodat het wat code bespaart:
```suggestion
        const { subregion, population, capital, money, currencies, name } = nameOfCountry.data[0];
        const money = currencies[0].name;
```